schema.sql

```diff
@@ -45,4 +45,10 @@ CREATE TABLE visits(
   vet_id INTEGER REFERENCES vets(id),
   animal_id INTEGER REFERENCES animal(id),
   date_of_visit DATE NOT NULL
-);
\ No newline at end of file
+);
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX email_asc ON owners (email ASC);
+
+CREATE INDEX vet_id on visits (vet_id ASC);
```

Comment:
Great job team 🤝

- [ ] [Optional]You may also create an index `CREATE INDEX ON visits (animal_id);`